pkg/providers/ovirt/client/ovirt-client.go

```diff
@@ -2,13 +2,23 @@ package ovirtclient
 
 import (
 	"fmt"
+	"time"
 
 	ovirtsdk "github.com/ovirt/go-ovirt"
 )
 
+const (
+	// Number of minutes to wait for VM to be stopped
+	vmStopTimeout = 5
```

Comment:
Can you add TODO to make them configurable? Maybe config? This can be done when needed not now.